### PR TITLE
use isorts black profile

### PR DESCRIPTION
**Pull request by CarliJoy**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Description

isort has a special black profile, this prevents that isorts changes a line and black changes it back.

See https://pycqa.github.io/isort/docs/configuration/black_compatibility.html

### CODE REVIEW

**Comment by CarliJoy** [comment]
@Pierre-Sassoulas not really a typo but close ;-)

File: setup.cfg [hunk 1]
```diff
@@ -76,9 +76,8 @@ markers =
 
 [isort]
 multi_line_output = 3
```
Comment (Pierre-Sassoulas) [inline setup.cfg:78]: I think this is also for black compatibility but I4ve done that a long time ago

```suggestion
```

**Review by Pierre-Sassoulas — APPROVED** [review]
Nice 👍

File: setup.cfg (hunk 1 above)
Comment (CarliJoy) [reply setup.cfg]: I tried it. Result of `pre-commit run --all-files` are the same, so I removed it as well...

**Comment by Pierre-Sassoulas** [comment]
Thank you, and congratulation on becoming a pylint contributor :) You're trusted by github action now 😉

**Pull request merged**